cli/src/main/resources/help/module.txt

```diff
@@ -39,7 +39,11 @@ DESCRIPTION
 
 ARGUMENTS
 
- --name          - (required) the name of the module to be added or removed.
+ --modules-path - (optional) The absolute file path to the server's modules 
```

Comment:
I saw the recommandation but thought that, being the root dir of the modules of a server, modules-path would be simpler for users to get what this option is. JBoss default location is "modules", and this option points to such "root location".
As David is suggesting, --module-root-dir is even more obvious.